### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by dalenewby**
### Changed
* Removed code to create Terraform module [terraform-aws-tfc-backup-to-b2](https://github.com/silinternational/terraform-aws-tfc-backup-to-b2).
* Replace `b2_fsbackup_repo` with `repo_string`.

### Added
* AWS policy to allow ecs:RunTask.

### CODE REVIEW

File: remote.tf [hunk 1]
```diff
@@ -15,7 +15,7 @@ data "terraform_remote_state" "b2-bucket" {
   config = {
     organization = "gtis"
     workspaces = {
-      name = "b2-${local.app_name_and_env}"
+      name = "b2-${var.app_name}-${var.app_env}"
```
Comment (briskt) [inline remote.tf:18]: Why are these reverting back to variables? Would you ever want `var.app_env` to be different than `data.terraform_remote_state.common.outputs.app_env`?

File: vars.tf [hunk 2]
```diff
@@ -10,6 +11,16 @@ variable "aws_secret_key" {
   default = null
 }
 
+variable "app_env" {
+  description = "The abbreviated app environment (e.g. prod or stg)"
+  type        = string
+}
+
+variable "app_environment" {
```
Comment (briskt) [inline vars.tf:19]: `app_environment` is  not used anywhere that I can see.

**Review by briskt — COMMENTED** [review]
A couple questions added.

File: remote.tf (hunk 1 above)
Comment (dalenewby) [reply remote.tf:18]: We destroyed `local.app_name_and_env`.  If someone else uses the Terraform module `terraform-aws-tfc-backup-to-b2`, they wouldn't want to be forced to use our `terraform_remote_state`.  They might prefer to set variables in Terraform Cloud.  Consider this code to be our example of "how we use `terraform-aws-tfc-backup-to-b2`.

File: vars.tf (hunk 2 above)
Comment (dalenewby) [reply vars.tf:19]: `app_environment` is used to create the `name_tag_suffix` in `terraform-aws-tfc-backup-to-b2`.

File: remote.tf (hunk 1 above)
Comment (briskt) [reply remote.tf:18]: Then why not give better flexibility and do:

File: vars.tf (hunk 2 above)
Comment (briskt) [reply vars.tf:19]: But in THIS module, you're not using `app_environment`.

File: README.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ data on a scheduled basis.
 * `b2_account_key` - Backblaze Application Key secret (`applicationKey`)
 * `b2_fsbackup_args` - Additional arguments to the `restic backup` command, default: ""
 * `b2_fsbackup_forget_args` - Arguments to the `restic forget` command, default: `--keep-daily 45`
-* `b2_fsbackup_repo` - Restic repository name, e.g., `b2:bucketname:directory`
+* `repo_string` - Restic repository name, e.g., `b2:bucketname:directory`
```
Comment (briskt) [inline README.md:26]: `repo_string` is not used anywhere

File: remote.tf [hunk 4]
```diff
@@ -15,7 +15,7 @@ data "terraform_remote_state" "b2-bucket" {
   config = {
     organization = "gtis"
```
Comment (briskt) [inline remote.tf:16]:
```suggestion
    organization = var.tf_remote_common_organization
```

**Review by briskt — APPROVED** [review]
Looks better. FYI, `repo_string` is still listed in the README.

**Pull request merged**